modules/luci-mod-status/root/usr/libexec/syslog-wrapper

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+. /lib/functions.sh
+
+main() {
+	local log_file
+
+	log_file="$(uci_get system @system[0] log_file "")"
+	if [ -f "$log_file" ]; then
+		cat "$log_file"
+		return
```

Comment:
A few points: `cat $file` and then `return`. This effectively means that `cat` return code is bubbled up from the script, right? Why do we prefer `cat` when we can do, e.g. `logread -F`? Can you test with that? Then we remain with the same tools and facilities (no risk of different return codes being passed).

Reply:
Sorry, but English is not my native language. The translation of your text suggests "bubble up" in the context of filter something. 
But the mere translation of "bubble up" means something like inflate.

I will show you the output of `cat file`:
![grafik](https://github.com/openwrt/luci/assets/64612743/b08be32b-e669-48b0-9a63-d217e4f7ad56)

Unlike `logread` this enables us to show older log messages than from the time booted.

`logread -F` does not return something but writes the content of it to the file specified after `-F`. I already tried it in the creation process, because I thought like you that the function would be cleaner.

Reply:
Is something left to do for me?
Or can it be pulled to master?